messaging/kafka-avro-reactive-messaging/src/main/resources/application.properties

```diff
@@ -2,17 +2,23 @@
 %dev.kafka.bootstrap.servers=localhost:9092
 #TODO: Disable health check because of https://github.com/quarkusio/quarkus/issues/15464
 %dev.quarkus.reactive-messaging.health.enabled=false
+
+%dev.cron.expr=*/1 * * * * ?
+%dev.mp.messaging.outgoing.source-stock-price.merge=true
+
 %dev.mp.messaging.connector.smallrye-kafka.apicurio.registry.url=http://localhost:8081/apis/registry/v2
+
 %dev.mp.messaging.outgoing.source-stock-price.connector=smallrye-kafka
 %dev.mp.messaging.outgoing.source-stock-price.topic=stock-price
 %dev.mp.messaging.outgoing.source-stock-price.value.serializer=io.apicurio.registry.serde.avro.AvroKafkaSerializer
 %dev.mp.messaging.outgoing.source-stock-price.apicurio.registry.avro-datum-provider=io.apicurio.registry.serde.avro.ReflectAvroDatumProvider
 %dev.mp.messaging.outgoing.source-stock-price.apicurio.registry.auto-register=true
 
+%dev.mp.messaging.incoming.channel-stock-price.group.id=${quarkus.uuid}
 %dev.mp.messaging.incoming.channel-stock-price.connector=smallrye-kafka
 %dev.mp.messaging.incoming.channel-stock-price.topic=stock-price
-%dev.mp.messaging.incoming.channel-stock-price.specific.avro.reader=true
+#%dev.mp.messaging.incoming.channel-stock-price.specific.avro.reader=true
```

Comment:
Because I was reviewing our properties and I didn't found proper documentation about this config.
- Quarkus conf: https://quarkus.io/guides/all-config (not found)
- https://quarkus.io/guides/kafka-schema-registry-avro#the-movie-consumer
```
We don’t have to define the apicurio.registry.use-specific-avro-reader property either, that is also configured automatically.
f you use Confluent Schema Registry, you don’t have to configure value.deserializer or specific.avro.reader either. They are both detected automatically.
```
- https://smallrye.io/smallrye-reactive-messaging/smallrye-reactive-messaging/2.1/kafka/kafka.html#kafka-avro-configuration   

So, I prefer to remove based on the data that I have (0 magic).